WordPress/twentynineteen
# Posts - More link: replace angular quotation mark with arrow

The angular quotation mark is a punctuation character in spanish so it looks odd to use it as a UI element. Despite the fact that the theme is in english originally, it's necessary to update this since this character is in the CSS and can't be translated.

This commit replaces it with an arrow, more appropriate for any language. Additionally, this PR also removes a duplicated `display` property.

### Before

<img width="785" alt="captura de pantalla 2018-10-27 a la s 21 34 19" src="https://user-images.githubusercontent.com/1041600/47610646-ee885e80-da30-11e8-9adb-608481da57bc.png">

### After

<img width="779" alt="captura de pantalla 2018-10-27 a la s 21 33 58" src="https://user-images.githubusercontent.com/1041600/47610645-ee885e80-da30-11e8-9426-1347526d6f59.png">

### Testing instructions

1. Checkout this branch and build with `npm install && npm run build:style`
2. add a new post
3. add a paragraph, then the More separator, and then another paragraph
    <img width="651" alt="captura de pantalla 2018-10-27 a la s 21 45 23" src="https://user-images.githubusercontent.com/1041600/47610672-b7ff1380-da31-11e8-8c13-6615d26c95eb.png">
4. publish and check in post list view
